pkg/skaffold/schema/latest/config.go

```diff
@@ -98,6 +101,13 @@ type PortForwardResource struct {
 	LocalPort int `yaml:"localPort,omitempty"`
 }
 
+// DebugConfig contains global configuration for `skaffold debug`.
+type DebugConfig struct {
+	// DuctTapeRepo overrides the image repository for the debugging duct-tape init images.
+	// Defaults to gcr.io/gcp-dev-tools/duct-tape .
```

Comment:
```suggestion
	// Defaults to `gcr.io/gcp-dev-tools/duct-tape`.
```

Will set the right default value in the yaml schema